wger/urls.py

```diff
@@ -118,6 +118,11 @@
     r'setting-weightunit',
     core_api_views.WeightUnitViewSet,
     base_name='setting-weight-unit')
+router.register(
```

Comment:
I have tested these two endpoints and they give errors:
`http://localhost:8000/api/v2/user/`
![image](https://user-images.githubusercontent.com/8415776/31962720-038eea7a-b907-11e7-81b3-0d2016fe5496.png)
`http://localhost:8000/api/v2/apiusers/`
![image](https://user-images.githubusercontent.com/8415776/31962788-30ea874a-b907-11e7-9544-923b8ac69634.png)

Make sure these endpoints can onkly be accessed by only authenticated users